index.js

```diff
@@ -231,7 +231,18 @@ module.exports = function (content) {
     // indentedSyntax is a boolean flag
     opt.indentedSyntax = Boolean(opt.indentedSyntax);
 
-    opt.importer = getWebpackImporter();
+    // `node-sass`'s importer takes an array of functions. Since `parseQuery` doesn't allow functions,
+    // `opt.importer` has to be a path or array of paths that, once `require`d, return importer functions.
+    opt.importer = []
+        .concat(opt.importer || [])
```

Comment:
`opt.importer` may be a string (to mirror `node-sass`s `importer` field which takes a function or array of functions).